### PR TITLE
[COMPX-5169] Add QUEUEMANAGER service to ODS blueprint for 7.2.9

### PR DESCRIPTION
**Pull request by jgolieb**
Tested on mow-int.

### CODE REVIEW

File: core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp [hunk 1]
```diff
@@ -173,12 +173,12 @@
             "base": true
           },
           {
-              "refName": "hbase-HBASETHRIFTSERVER-BASE",
-              "roleType": "HBASETHRIFTSERVER",
-              "displayName": null,
-              "configs": [],
-              "base": true
-           },
+            "refName": "hbase-HBASETHRIFTSERVER-BASE",
```
Comment (doktoric) [inline core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp:176]: why is this changed?

File: core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp [hunk 2]
```diff
@@ -272,6 +272,28 @@
         ],
         "displayName": null,
         "roles": []
+      },
+      {
+        "refName": "queuemanager",
```
Comment (doktoric) [inline core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp:277]: did we tested this ?

File: core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp (hunk 1 above)
Comment (jgolieb) [reply core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp:176]: That's just spacing.

File: core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp (hunk 2 above)
Comment (jgolieb) [reply core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp:277]: Yes, as stated in the description, this was tested by deploying on a mow-int environment.

File: core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp (hunk 1 above)
Comment (doktoric) [reply core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp:176]: please revert back that part because this is not related to the change

Comment (jgolieb) [reply core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp]: done

**Review by doktoric — APPROVED** [review]
